content/docs/2.0/migration.md

```diff
@@ -55,10 +56,16 @@ spec:
   # {list of triggers to activate the deployment}
 ```
 
-
 ### Scaling of Jobs
 
-TODO description
+In order to scale `Jobs` with KEDA v2, you need to do only a few modifications to existing v1 `ScaledObjects` definitions, so they comply with v2:
+- Change the value of `apiVersion` property from `keda.k8s.io/v1alpha1` to `keda.sh/v1alpha1`
+- Change the value of `kind` property from `ScaledObject` to `ScaledJob`
+- Remove property `spec.scaleType`
+- Remove properties `spec.cooldownPeriod` and `spec.minReplicaCount`   
+- Rename property `spec.scaleTargetRef.deploymentName` to `spec.scaleTargetRef.name`
```

Comment:
`spec.scaleTargetRef.name` is not being used with ScaledJobs, am I right?

Reply:
Sorry, Forgot to remove. Done.